src/partials/_imports.scss

```diff
@@ -0,0 +1,4 @@
+//imports go here (google fonts, font awesome, etc
+@import url('https://fonts.googleapis.com/css2?family=DynaPuff:wght@500&family=Montserrat:wght@400;700&display=swap');
```

Comment:
I think we should use a link tag instead of importing the google font, it might marginally improve the performance of our site. You can drop the google link snippet in the public folder > index.html. 

Ref: https://stackoverflow.com/questions/12316501/including-google-fonts-link-or-import